squarelet/organizations/admin.py

```diff
@@ -282,6 +366,28 @@ def get_email_domains(self, obj):
 
     get_email_domains.short_description = "Email Domains"
 
+    def get_outstanding_invoices(self, obj):
+        """Display count and total of outstanding invoices"""
+        count = getattr(obj, "outstanding_invoice_count", 0)
+        total = getattr(obj, "outstanding_invoice_total", 0)
+        if count > 0:
+            return f"{count} (${total / 100:.2f})"
+        return "-"
+
+    get_outstanding_invoices.short_description = "Outstanding Invoices"
+    get_outstanding_invoices.admin_order_field = "outstanding_invoice_count"
+
+    def is_visible_to_anonymous(self, obj):
```

Comment:
This is adding a query per organization the the list display page and may affect performance

Reply:
I'm fine removing this column, since we have another admin view for seeing invoices.